src/Components/Header/Header.js

```diff
@@ -54,7 +58,7 @@ const Header = () => {
       <Paper className="header-full-width-container" square={true} elevation={0}>
         <AppBar id="nav-container" position="sticky" elevation={0}>
           <a href={`/step-1${queryString}`} className="home-link">
-            <img src={getReferrer('logoSource')} alt={getReferrer('logoAlt')} className={getReferrer('logoClass')} />
+            <img src={getLogo(headerLogoByState)} alt={getReferrer('logoAlt')} className={getReferrer('logoClass')} />
```

Comment:
We use `getRefferrer` to use a white label logo for specific referrers. For example, you will see a different logo at https://benefits-calculator-staging.herokuapp.com/step-1?referrer=bia. This pr removes that functionality. What we need to do is change the default logo in `referrerHook.ts`.